src/cluster_asm.c

```diff
@@ -2634,3 +2752,200 @@ void trimslotsCommand(client *c) {
     slotRangeArrayFree(slots);
     addReply(c, shared.ok);
 }
+
+/* Start the active trim job. */
+void asmActiveTrimStart(void) {
+    if (listLength(asmManager->active_trim_jobs) != 1) return;
+
+    slotRangeArray *slots = listNodeValue(listFirst(asmManager->active_trim_jobs));
+
+    asmManager->active_trim_started++;
+    asmManager->active_trim_keys_deleted = 0;
+
+    /* Count the number of keys to trim */
+    for (int i = 0; i < slots->num_ranges; i++)
+        for (int slot = slots->ranges[i].start; slot <= slots->ranges[i].end; slot++)
+            asmManager->active_trim_keys_total += kvstoreDictSize(server.db[0].keys, slot);
```

Comment:
`{}` may be much clear
```suggestion
    for (int i = 0; i < slots->num_ranges; i++)  {
        for (int slot = slots->ranges[i].start; slot <= slots->ranges[i].end; slot++)
            asmManager->active_trim_keys_total += kvstoreDictSize(server.db[0].keys, slot);
    }        
```